src/core/webidl.js

```diff
@@ -612,17 +612,37 @@ function findDfn(parent, name, definitionMap, type, idlElem) {
   const originalName = name;
   parent = parent.toLowerCase();
   switch (type) {
+    case "attribute": {
+      const asLocalName = name.toLowerCase();
+      const asQualifiedName = parent + "." + asLocalName;
+      let dfn;
+      if (definitionMap[asQualifiedName] || definitionMap[asLocalName]) {
+        dfn = findDfn(parent, asLocalName, definitionMap, null, idlElem);
+      }
+      if (!dfn) {
+        break; // try finding dfn using name, using normal search path...
+      }
+      const lt = dfn[0].dataset.lt ? dfn[0].dataset.lt.split("|") : [];
+      lt.push(asQualifiedName, asLocalName);
+      dfn[0].dataset.lt = [...new Set(lt)].join("|");
+      return dfn;
+    }
     case "operation": {
       // Overloads all have unique names
       if (name.search("!overload") !== -1) {
         name = name.toLowerCase();
         break;
       }
       // Allow linking to both "method()" and "method" name.
-      const asMethodName = name.toLowerCase() + "()";
-      const asFullyQualifiedName = parent + "." + name.toLowerCase() + "()";
+      const asLocalName = name.toLowerCase();
+      const asMethodName = asLocalName + "()";
+      const asQualifiedName = parent + "." + asLocalName;
+      const asFullyQualifiedName = asQualifiedName + "()";
 
-      if (definitionMap[asMethodName] || definitionMap[asFullyQualifiedName]) {
+      if (
+        definitionMap[asMethodName] ||
+        definitionMap[asFullyQualifiedName.toLowerCase()]
```

Comment:
Are we sure we need a `.toLowerCase()` here?

Reply:
yeah, definitely. everything in the definition map is lowercase.

Reply:
(eventually we should have a idlDefinitionMap, which is case sensitive)

Reply:
Oh yes. I missed we haven't done it yet.